analytics/articles.js

```diff
@@ -72,6 +72,14 @@ if(location.pathname.match('/article/')){
     });
   });
 
+  analyticsHooks.on('impressions:editorial-signup', function(options){
+    analytics.track('Viewed popup for editorial email', {
```

Comment:
I believe these are now in the style, please let me know if anything looks amiss!